docs/sources/getting-started/build-first-dashboard.md

```diff
@@ -43,18 +43,19 @@ We strongly recommend that you change the default administrator password.
 
 If you've already set up a data source that you know how to query, refer to [Create a dashboard]({{< relref "../dashboards/build-dashboards/create-dashboard" >}}) instead.
 
-To create your first dashboard using the built-in `Grafana` data source:
+To create your first dashboard using the built-in `-- Grafana --` data source:
```

Comment:
Do we need to rename `Grafana` to `--Grafana--` [here](https://github.com/grafana/grafana/pull/69280/files#diff-8d40b3d5b3af0206f8187b33f8bc3a07da8949536bc5f1bc687a93455299a914R13)?